Makefile

```diff
@@ -46,6 +46,16 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+.PHONY: code/check
+code/check:
```

Comment:
Never convinced by these phony make targets using file paths. We haven't done this for any others, so personally i woudln't start doing it now for these. You might also want to take inspiration from how we have setup individual make files in other projects like here https://github.com/Kuadrant/kuadrant-operator/blob/main/make/verify.mk

Reply:
Would we want to extract formatting and vendor to another folder? 🤔 The makefile right now looks segmented with `General`, `Build`, `Deployment` and `Build Dependencies` sections. I could go further and extract those into separate files  as they tend to grow immensely 🤔  WDYT?

Reply:
Closely followed the provided example :thinking:

Reply:
I think the main thing here is that we just don't deviate from the naming that we already are using, so verify-* targets would make more sense to me. If we ere actually using make to manage files then sure file paths obviously make sense. 

Individual Makefiles, it's really if it makes sense. The Makefile can get quite unwieldy if we just keep adding loads of targets, if we have several verify-* type targets that are specifically used to verify stuff sis an expected state and we plan to run thos eon CI, maybe they could be grouped together somewhere.